microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -106,11 +105,11 @@ The information provided in the following section is displayed to users who revi
 | Small promotional tile \(optional\) | The `Small promotional tile` image is used to display your extension alongside other extensions in the store.  The size of the image should be 440 x 280 pixels.  Additionally, you may choose to copy the asset from one language to all other languages using the duplicate button.  The button is found following the field after you upload a promotional tile for the language.  |  
 | Screenshots \(optional\) | You may submit a maximum of 10 `screenshots` describing the functionality of your extension in detail.  The size of the screenshots must be either 640 x 480 pixels or 1280 x 800 pixels.  Additionally, you may choose to copy the asset from one language to all other languages using the duplicate button.  The button is found following the field after you upload at least one for the language.|  
 | Large promotional tile \(optional\) | `Large promotion tiles` are used in the store to feature extensions more prominently in the Microsoft Edge add-ons website.  The images, if submitted, are visible to the users.  The size of the PNG files must be 1400 x 560 pixels.  Additionally, you may choose to copy the asset from one language to all other languages using the duplicate button.  The button is found following the field after you upload a promotional tile for the language.  |  
-| YouTube video URL \(optional\) | You may include a promotional YouTube video of your extension.  The `YouTube video URL` video is displayed on the store listing page of your extension.  |  
+| YouTube video URL \(optional\) | You may include a promotional YouTube video of your extension.  The `YouTube video URL` video is displayed on the store listing webpage of your extension.  |  
 | Short description \(required\) | To edit the `short description`, you must update the description field in your manifest file of your extension package and reupload it.  |  
-| Search terms \(optional\) | `Search terms` are single words or phrases that help users discover your extension when searching in the Microsoft Edge Add-ons store.  The search terms aren't displayed to users.  |  
+| Search terms \(optional\) | `Search terms` are single words or phrases that help discover your extension a user searches in the Microsoft Edge Add-ons store.  The search terms aren't displayed to users.  |  
```

Comment:
Change:
single words or phrases that help discover your extension a user searches in the Microsoft Edge Add-ons store
to:
single words or phrases that help discover your extension when a user searches in the Microsoft Edge Add-ons store
or revert to the original wording.

Reply:
Updated.